src/Helper/SerializationHelper.php

```diff
@@ -185,4 +188,15 @@ public static function toSnakeCase($input)
         }
         return implode('_', $ret);
     }
+
+    /**
+     * Check for multiple types of invalid values in a variable and returns.
+     *
+     * @param $var
+     * @return bool
+     */
+    public static function isNull($var)
+    {
+        return (!$var && $var !== 0 && $var !== '0' && $var !== false && $var !== 0.00);
```

Comment:
I'm unsure about this. @hannunosto might know more but just sanity check this please.

Reply:
@mridang This check existed already, we just wrapped into a method for reuse purposes:
https://github.com/Nosto/nosto-php-sdk/blob/master/src/Helper/HtmlMarkupSerializationHelper.php#L97